cachecontrol/caches/file_cache.py

```diff
@@ -101,7 +101,7 @@ def get(self, key):
         with open(name, 'rb') as fh:
             return fh.read()
 
-    def set(self, key, value):
+    def set(self, key, value, expires=None):
```

Comment:
It would be worth adding a comment that the file cache doesn't use the expires argument and its inclusion in the API is to support data stores that provide expiration. The file cache is used pretty heavily, so making a note for future readers would be appreciated.